robot_skills/test/test_audio.py

```diff
@@ -1,33 +1,24 @@
 #! /usr/bin/env python
-import roslib; roslib.load_manifest('robot_skills')
 
+# System
 import random
-import time
-
 import sys
+import time
 
+# ROS
 import rospy
 
+# TU/e Robotics
+from robot_skills import get_robot
+
 if len(sys.argv) < 2:
```

Comment:
Add argument check like other files to be consistent

Reply:
See a71172c306547eba6d50c057baaef8c0ec10a1d0 and 106535870505f785620744139c45820da4f330a4